addons/website/static/src/js/content/website_root.js

```diff
@@ -223,6 +219,22 @@ var WebsiteRoot = publicRootData.PublicRoot.extend(KeyboardNavigationMixin, {
         ev.data.options.editableMode = ev.data.editableMode;
         this._super.apply(this, arguments);
     },
+    /**
+     *
+     * @private
+     * @param {*} ev
+     */
+    _onAdjustIFrame: function(ev) {
+        let adjustIframe = true;
+        const resizeFunction = () => {
+            window.dispatchEvent(new Event('resize'));
+            if (adjustIframe) {
+                window.requestAnimationFrame(resizeFunction);
+                adjustIframe = false;
+            }
+        };
+        window.requestAnimationFrame(resizeFunction);
+    },
```

Comment:
This implementation seems strange, a version without the `adjustIframe` variable would surely be better... but given my previous comment about the `resizing` variable, this is probably not ok anyway.